openmetadata-spec/src/main/resources/json/schema/entity/services/connections/database/athenaConnection.json

```diff
@@ -85,6 +85,10 @@
       "description": "Supports Lineage Extraction.",
       "type": "boolean",
       "default": true
+    },
+    "sampleDataStorageConfig": {
+      "title": "S3 Config for Sample Data",
+      "$ref": "../connectionBasicType.json#/definitions/sampleDataStorageConfig"
```

Comment:
should the services have only the sampleDataStorageConfig or the overall ProfilerConfig?

Reply:
Ummm I think we only discussed to add the storage config, do we want to add the profiler config here?

Reply:
ah ok then I might be remembering it wrong. If that's what we agreed then we can move on cc @TeddyCr